frontend/src/i18n/index.ts

```diff
@@ -38,6 +38,11 @@ const defaultShortDatetimeFormat = {
     year: 'numeric',
     month: 'long',
     day: 'numeric'
+  },
+  daytime:{
+    hour: '2-digit', 
+    minute: '2-digit', 
+    second: '2-digit' 
```

Comment:
Would make a `long` formatter that includes time and date, then you can call `{{ d(device.creationTime, 'long') }}` instead of `{{ d(device.creationTime, 'short') }} {{ d(device.creationTime, 'daytime') }}`